app/api/items/router.py

```diff
@@ -1,11 +1,20 @@
 from fastapi import APIRouter, Depends
-
-from .dependencies import validate_item_id
-from .schemas import Item
+from sqlalchemy.orm import Session
+from .dependencies import validate_item_id, get_db
+from .schemas import Item, ItemPhoto
+from .service import get_photo_by_item_id
+from .exceptions import PhotoNotFound
 
 router = APIRouter()
 
 
 @router.get("/{item_id}", response_model=Item)
 def read_item(item: Item = Depends(validate_item_id)):
     return item
+
+
+@router.get("/{item_id}/photos", response_model=ItemPhoto)
+def read_photos(item: Item = Depends(validate_item_id), db: Session = Depends(get_db)):
+    photos_id = get_photo_by_item_id(db, item.id)
+
+    return photos_id
```

Comment:
```suggestion
    photo_ids = get_photos_by_item_id(db, item.id)

    return photo_ids
```

是拿到多個「照片的 ID」，不是「多個照片」的 ID